### DISCUSSION TITLE
Some Button in WebUI should have double confirmation mechanism

### DISCUSSION THREAD

**Discussion by kuolemaaa** [arendst/Tasmota#22124] [Ideas]
I got a smart plug with ESP8266 and Tasmota in it and IM LOVING IT. 

The problem is the Toggle button in the WebUI to turn it ON and OFF. 

Since this smart plug could be used with sort-of critical system (like a computer, or any other item that should not be abruptly turned off) I think it is a good idea that some button (like the on/off toggle) should have some sort of double confirmation just to avoid to turn something on or off with a misclick or a mis-tap/touch

**Comment by sfromis** [comment]
The rudimentary web UI is just not meant for "daily operation", which is expected to be via a backend system, where you can manage special cases like this.

One possible way local on the device could be to use `PowerLock` to disable power on/off, and then a second virtual relay with rules to enable or disable `PowerLock`, where you could also set a `PulseTime` to have it lock again after a couple of seconds.
